src/style.css

```diff
@@ -0,0 +1,93 @@
+@import url(./styles/reset.css);
+
+.html {
+  font-family: Arial, Helvetica, sans-serif;
+}
+
+.blocks {
+  display: flex;
+  flex-wrap: wrap;
+}
+
+.blocks__block {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 300px;
+  min-width: 300px;
+  font-size: 100px;
+  color: white;
+  flex-grow: 1;
+  flex-basis: 33.33%;
+}
+
+.blocks__block--1 {
+  background-color: rgb(100% 0% 0%);
```

Comment:
in rgb you need to indicate not percent, but a number. that's why deployed version doesn't show  colors

Reply:
For me, it's very strange, because in my local project it works correctly.